doc/website/release-notes/iceoryx-v2-0-6.md

```diff
@@ -0,0 +1,11 @@
+# iceoryx v2.0.6
```

Comment:
@elBoberido Please have a look at this PR: https://github.com/eclipse-iceoryx/iceoryx/pull/2126/files

You need to make this file available in the MkDocs navigation by adding it to `doc/website/release-notes/.pages`